share/revng/test/tests/analysis/ImportFromCAnalysis/rft-multiple-regs-for-return-val.model.yml.ccode

```diff
@@ -3,7 +3,7 @@
 //
 
 _ABI(raw_x86_64)
-struct return_value {
```

Comment:
Actuall, size on the return value of a RFT doesn't make sense, because these aren't "real" types that live in memory, but they are only "bags of registers".

@aleclearmind any thought?

Reply:
Actually, considering that the consensus seems to be for `_SIZE(X)` to be optional, there is indeed no reason to add this here.

Reply:
This is gone.